src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -56,15 +56,11 @@ var validUTCDateTime = function (dateTimeComposition){
 var validString = function (dateTimeString) {
```

Comment:
Could you change the name to something like 'validDateString'? This method is now testing specifically for date formats so the name should reflect that.

Reply:
Thanks~ It's a good suggestion after my changes. I have renamed the function name.